scripts/cluster/images.py

```diff
@@ -25,5 +30,32 @@ def import_images(image: str):
     do_image_import(image_data)
 
 
+def get_all_ctr_images():
+    """
+    Return list of all OCI images from containerd.
+    """
+    images = subprocess.check_output([CTR, "image", "ls", "--quiet"]).decode().split("\n")
+
+    # drop the sha256-digest aliases
+    return [tag for tag in images if tag and not tag.startswith("sha256:")]
+
+
+@images.command("export", help="Export OCI images from the current MicroK8s node")
```

Comment:
The export is not the opposite of import so could we rename the command to something like `export-local`. This would make it easier for us to provide an `export` command in the future.